src/Config/ConfigurationLoader.php

```diff
@@ -109,11 +109,11 @@ function ($json) {
             if ($indexer->hasBandits()) {
                 $this->fetchBanditsAsRequired($indexer);
             }
-        }
 
-        // Store metadata for next time.
-        $this->configurationStore->setMetadata(self::KEY_FLAG_TIMESTAMP, $this->millitime());
-        $this->configurationStore->setMetadata(self::KEY_FLAG_ETAG, $response->ETag);
+            // Store metadata for next time.
```

Comment:
key change: move this set inside the `if modified` block

Reply:
Makes sense!